src/reporter/reporter.py

```diff
@@ -147,15 +147,26 @@ def notify():
         fiware_sp = request.headers.get('fiware-servicepath', None)
     else:
         fiware_sp = None
-
-    # Send valid entities to translator
-    with translator_for(fiware_s) as trans:
-        trans.insert(payload, fiware_s, fiware_sp)
-
-    msg = 'Notification successfully processed'
+    for entity in payload:
+        attrs = list(iter_entity_attrs(entity))
+    attrs.pop()
```

Comment:
why pop here? sorry I'm probably missing something...

Reply:
Hi @c0c0n3 , I have used attrs.pop() to filter out time_index value from attrs.

Reply:
Ha! That guy! I totally forgot about that. But in general, we shouldn't assume `time_index` comes first... Anyhoo, all this shouldn't be an issue if put the new code into that `_filter_empty_entities` method I mentioned earlier and then call `_filter_empty_entities` **before** the `time_index` gets added---see my suggestion above.